### ISSUE TITLE
[Bug-App]: Default system log x-axis time unit convertion not working

### ISSUE THREAD

**Issue by Dipan-Zhang** [wandb/wandb#10897]
### Describe the bug

# Summary
When a wandb run lasts over 600 seconds, the UI automatically converts units to minutes; however, the actual number is not converted, causing the plot to be smashed on the very left side.

Plot before 600s

<img width="1438" height="388" alt="Image" src="https://github.com/user-attachments/assets/b9deef3f-5cb9-4b1a-8557-b7310d0fcb9e" />

Plot after 600s:

<img width="1461" height="377" alt="Image" src="https://github.com/user-attachments/assets/9fa3c6c2-f98e-4a64-9e5e-5e3d2295fb70" />

## System Overview
system: Ubuntu 22.04
wandb version: tried 0.17.0 and 0.24.0, both not working
logging code:
```Python

      wandb.init(
          id=wandb_run_id,
          project=project,
          entity=entity,
          name=wandb_job_name,
          notes=cfg.get("wandb", {}).get("notes"),
          tags=cfg_to_group(cfg, return_list=True),
          dir=log_dir,
          config=OmegaConf.to_container(cfg, resolve=True),
          save_code=False,
          job_type="train_eval",
      )
      print(colored("Logs will be synced with wandb.", "blue", attrs=["bold"]))
      logging.info(f"Track this run --> {colored(wandb.run.get_url(), 'yellow', attrs=['bold'])}")
      self._wandb = wandb



    def log_dict(self, d:dict, step:int, mode:str="train"):
        assert mode in {"train", "eval"}
        if self._wandb is not None:
            for k, v in d.items():
                if not isinstance(v, (int, float, str)):
                    logging.warning(
                        f'WandB logging of key "{k}" was ignored as its type is not handled by this wrapper.'
                    )
                    continue
                self._wandb.log({f"{mode}/{k}": v}, step=step)

=====================
def log_train_info(logger: Logger, info, step, cfg, dataset, is_online):
    loss = info["loss"]
    grad_norm = info["grad_norm"]
    lr = info["lr"]
    update_s = info["update_s"]
    dataloading_s = info["dataloading_s"]

    # A sample is an (observation,action) pair, where observation and action
    # can be on multiple timestamps. In a batch, we have `batch_size`` number of samples.
    num_samples = (step + 1) * cfg.training.batch_size
    avg_samples_per_ep = dataset.num_samples / dataset.num_episodes
    num_episodes = num_samples / avg_samples_per_ep
    num_epochs = num_samples / dataset.num_samples
    log_items = [
        f"step:{format_big_number(step)}",
        # number of samples seen during training
        f"smpl:{format_big_number(num_samples)}",
        # number of episodes seen during training
        f"ep:{format_big_number(num_episodes)}",
        # number of time all unique samples are seen
        f"epch:{num_epochs:.2f}",
        f"loss:{loss:.3f}",
        f"grdn:{grad_norm:.3f}",
        f"lr:{lr:0.1e}",
        # in seconds
        f"updt_s:{update_s:.3f}",
        # if not ~0, you are bottlenecked by cpu or io
        f"data_s:{dataloading_s:.3f}",
    ]
    logging.info(" ".join(log_items))

    info["step"] = step
    info["num_samples"] = num_samples
    info["num_episodes"] = num_episodes
    info["num_epochs"] = num_epochs
    info["is_online"] = is_online

    logger.log_dict(info, step, mode="train")

```

**Comment by Dipan-Zhang** [comment]
Thanks for replying! Yeah, when I switched to random sampling, the plot showed as expected.

**Comment by Dipan-Zhang** [comment]
Hi Bonnie Shen,
Yeah, I can confirm now full Fidelity works now for me as well. Thanks again for fixing this bug!
